flows/aggregate.py

```diff
@@ -539,7 +539,7 @@ def parameters(batch: Sequence[DocumentStem]) -> dict[str, Any]:
         counter=n_batches,
         batches=batches,
         parameters=parameters,
-        unwrap_result=True,
+        unwrap_result=False,
```

Comment:
This fails now we have fixed `map_as_sub_flow ` to await the async tasks. We don't need the result from the sub deployment so let's just not unwrap. 

Alternatively we could persist results from the sub deployments of `aggregate_batch_of_documents` but again we don't actually need the result.